src/App.js

```diff
@@ -1,42 +1,134 @@
+        }
+      };
+      const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+      
+      console.log(url);
+      console.log("test",options);
+
+
+        try {
+          const response = await fetch(url, options);
+
+          if (!response.ok) {
+          const message = `Error: ${response.status}`;
+          throw new Error(message);
+          }
+        const data = await response.json();
+
+      console.log("DATA",data);
+
+      const todos = data.records.map((todo) => {
+
+        const newTodo = {
+          id: todo.id,
+          title: todo.fields.title
+        }
+        return newTodo
+      });
+        
+      setTodoList(todos);
+      setIsLoading(false);
+     
+      // console.log(data);
+      return data;
+        }
+        catch {
+          console.log("error messange");
+      };
+  
+    };
+
+    const postData = async(newTodo) => {
+
+      const options = {
+        method: "POST",
+        headers: {
+          Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+          "Content-Type": "application/json"
+        }, 
+        body: JSON.stringify(
+          {
+            fields: {
+              title: newTodo.title
+            }
+          }
+        )
+    
+      }
+      const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+      
+      console.log(url);
+      console.log(options);
+    
+    
+      try {
+        
+        
+        const response = await fetch(url, options);
+    
+          if (!response.ok) {
+            const message = `Error: ${response.status}`;
+                 throw new Error(message);
+          }
+          const data = await response.json();
+              const newTodo = {
+              id: data.id,
+              title: data.fields.title
+            }
+          setTodoList([...todoList, newTodo]);     
+          setIsLoading(false);
+          
+          console.log(data);
+          return data;
+    
+      }
+      catch{
+        console.log("error message")
+      };
+    
+    }
   
   React.useEffect(
     () => {
-      new Promise((resolve) =>
-      setTimeout(
-        () => resolve({ data: {todoList: JSON.parse(localStorage.getItem('savedTodoList'))} }),
-        2000
-      )
-  
-      ).then((result) => {setTodoList(result.data.todoList); setIsLoading(false);})
+     fetchData();
     },
     []
-  )
+  );
 
 
   React.useEffect(() => {
-       if(!isLoading) {
+       
+    if(!isLoading) {
         localStorage.setItem('savedTodoList', JSON.stringify(todoList));
        }
     
       }, [todoList]); 
 
   const addTodo = (newTodo) => {
-    setTodoList([...todoList, newTodo]);
+    postData(newTodo);
   }
 
 
-const removeTodo = (id) => {
+  const removeTodo = (id) => {
   const newremoveTodo = todoList.filter(
```

Comment:
I think I missed in 1.6 that the `newRemoveTodo` variable name is a little bit confusing. This array will be the modified copy of the array, which has had the deleted item filtered out, so let's try and find a name that indicates that more clearly. 

Maybe something like `filteredTodoList` or `nextTodoList`?

```suggestion
  const filteredTodoList = todoList.filter(
```

Reply:
Fixed